linter.py

```diff
@@ -103,10 +105,10 @@ def xo_fix(self, view, content):
 	if encoding == 'Undefined':
 		encoding = 'utf-8'
 
-	print('xo_fix -> content length:', len(content))
-	print('xo_fix -> encoding:', encoding)
-	print('xo_fix -> xo_env.PWD:', self.xo_env['PWD'])
-	print('xo_fix -> xo_project_root:', self.xo_project_root)
+	logger.info('xo_fix -> content length: %d', len(content))
```

Comment:
```suggestion
	logger.info('xo_fix → content length: %d', len(content))
```

Since we are already doing changes here. How about we use a real arrow.

Reply:
Hahah, good one.